### PR TITLE
feat: Enhance show views

### LINKED ISSUE #15011: Feature: Enhance show views
**Summary**

After https://github.com/datafuselabs/databend/issues/14889 finished. We can enhance `show views`. 

Display like this:

https://docs.snowflake.com/en/sql-reference/sql/show-views

And we need the most important display field is `text`.

> text | The text of the command that created the view (e.g. CREATE VIEW …).

### PR DESCRIPTION
**Pull request by blackstar-baba**
I hereby agree to the terms of the CLA available at: https://docs.databend.com/dev/policies/cla/

## Summary

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

- Fixes #15011 

## Tests

- [ ] Unit Test
- [ ] Logic Test
- [ ] Benchmark Test
- [x] No Test  - _Explain why_

## Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

### CODE REVIEW

**Comment by blackstar-baba** [comment]
how to test this modification? i can't find a example.

**Comment by TCeason** [comment]
> how to test this modification? i can't find a example.

You can fix the ci first.

**Comment by TCeason** [comment]
Please fix         FAIL [   1.211s] databend-query::it storages::system::test_columns_table

**Comment by TCeason** [comment]
> how to test this modification? i can't find a example.

In tests/sqllogictests/suites/base/06_show/06_0023_show_views.test , show views as least need display two field: name, view_query,

create view v1 as select * from default.t;
show views;
| name | view_query |
|--|--|
| v1 | select * from default.t |

File: src/query/sql/src/planner/binder/ddl/view.rs [hunk 1]
```diff
@@ -180,7 +180,8 @@ impl Binder {
                 .with_column("num_rows")
                 .with_column("data_size")
                 .with_column("data_compressed_size")
-                .with_column("index_size");
+                .with_column("index_size")
+                .with_column("view_query");
         } else {
             select_builder.with_column(format!("name AS `Views_in_{database}`"));
```
Comment (TCeason) [inline src/query/sql/src/planner/binder/ddl/view.rs:186]: if not full, also need with_column("view_query");

**Review by TCeason — APPROVED** [review]
LGTM.

**Comment by BohuTANG** [comment]
@blackstar-baba Thanks for your contribution!

**Pull request merged**